src/lib.rs

```diff
@@ -76,6 +76,25 @@ pub enum FeatureLevel {
     L12_1 = d3dcommon::D3D_FEATURE_LEVEL_12_1,
 }
 
+impl TryFrom<u32> for FeatureLevel {
```

Comment:
we shouid probably make this enum non-exhaustive, to support adding new variants later